examples/it.xsemantics.example.fj/src/it/xsemantics/example/fj/typing/fj-separated.xsemantics

```diff
@@ -33,6 +33,17 @@ import org.eclipse.xtext.EcoreUtil2
  */
 system it.xsemantics.example.fj.typing.FjSepTypeSystem extends FjTypeSystem
 
+copyright
```

Comment:
@mor-n4 sorry! I did not notice that that is valid Xsemantics syntax for copyright :D